### DISCUSSION TITLE
QQ: Can we have multiple resolvers pointing to the same entity?

### DISCUSSION THREAD

**Discussion by jtushman** [MichalLytek/type-graphql#821] [Q&A] [answered]
```ts
@Resolver(of => Recipe)
class TunaFishRecipeResolver {
  @FieldResolver(...)
  tunafishRecipe() {
    // ...
  }
}

@Resolver(of => Recipe)
class PeanutButterRecipeResolver {
  @FieldResolver(...)
  peanutbutterRecipe() {
    // ...
  }
}
```

(sorry for the contraived example)

any gotchas to this?

**Comment by MichalLytek** [comment, marked as answer]
Yes we can 😉
Why you haven't tried before asking a question?

**Reply by jtushman** [reply]
totally fair dig -- Michal -- but its a nice to have here in case others might have that question in the future.

You are the best -- type-graphql is the best

**Reply by andy-ex** [reply]
Hi @MichalLytek ! Is it possible to specify the dependency between resolvers so they are executed accordingly? In my case on resolver needs to use the result and be called after another (like a chain), but I do not see an opportunity to do so. Any help is appreciated!

**Reply by MichalLytek** [reply]
@andy-ex It's not possible - see #898
